cmd/kombiner-controller/init.go

```diff
@@ -3,5 +3,6 @@ package main
 import "flag"
 
 func init() {
-	flag.StringVar(&KubeConfig, "kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
```

Comment:
Defined through `sigs.k8s.io/controller-runtime.GetConfigOrDie()`

Reply:
Oh, it was on me. The config is required now. Nevermind.